### PR TITLE
Conversation API beta/beta2 endpoints transition to v0.1/v0.2

### PR DESCRIPTION
**Pull request by pardel**
# Changes 

* Transitioned the Conversation API endpoints from `beta` and `beta2` paths to `v0.1` and `v0.2` respectively.

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/conversation.yml [hunk 1]
```diff
@@ -201,7 +207,7 @@ paths:
         <div class="Vlt-callout Vlt-callout--warning">
         <i></i>
         <div class="Vlt-callout__content">
-        <p><strong>This API endpoint is deprecated. Please use <a href="/api/conversation.v2#get-events">/beta2/events</a></strong><br />This endpoint will return a maximum of 100 items.</p>
+        <p><strong>This API endpoint is deprecated. Please use <a href="/api/conversation.v2#get-events">/v0.2/events</a></strong><br />This endpoint will return a maximum of 100 items.</p>
         </div>
         </div>
```
[the remark this answers is not in the document]
Comment (lornajane) [reply definitions/conversation.yml:213]: Docs for deprecating a path item: http://spec.openapis.org/oas/v3.0.3.html#fixed-fields-7

**Review by lornajane — COMMENTED** [review]
I'm not ready to approve, sorry! I see that we're using HTML - I'd advise that markdown would be better here (I realise it's not in this PR, we can open an issue and follow up with another change if that's a better way to tackle it). Also the deprecated fields should be marked as deprecated as well as having this clear in their description.

File: definitions/conversation.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pardel) [reply definitions/conversation.yml]: I've added `deprecated` fields to all list endpoints. Happy to tackle the HTML in the upcoming v0.3 PR (will include lots of deprecation notices).

**Comment by pardel** [comment]
Added ticket to remove HTML descriptions: https://nexmoinc.atlassian.net/browse/DEVX-4681

**Review by lornajane — APPROVED** [review]
Looks sound to me!

**Pull request merged**